paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -226,6 +236,20 @@ public class PaparazziPlugin @Inject constructor(
 
         test.outputs.dir(reportOutputDir).withPropertyName("paparazzi.report.dir")
 
+        val isVerifying = isVerifyRun.map {
```

Comment:
It doesn't seem like we need this new field since it's always equal to `isVerifyRun`. Can we just extract the `if` check to set the test reporter, then revert the change on line 261?

Reply:
unfortunately you can't set the test reporter until isVerify is available. Doing it that way breaks config cache. I can add a comment about that